src/json/members/choiyoon.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Yoonjung Choi",
+    "pennkey": "choiyoon",
+    "major": "Undecided",
+    "school": "SAS",
+    "bio": "Hi, this is Yoonjung. I'm a BizDev for Penn Clubs.",
+    "hometown": "Hong Kong",
+    "team": "Penn Clubs",
+    "roles": ["Business Developer"],
+    "photo": "https://media.licdn.com/dms/image/D5603AQHYyI1WDPwFvw/profile-displayphoto-shrink_400_400/0/1678278503421?e=1701302400&v=beta&t=rGoJZl7lZCxJwDz3xJPJnMk4IbHkzY85eyXdMnB1mZ8",
+    "linkedin": "https://www.linkedin.com/in/yoonjungchoi05/",
+    "website": "",
+    "github": "https://github.com/yoonjung27",
+    "semester_joined": "2023F",
```

Comment:
Rather than saying 2023F, please reference our guide here on notation: https://www.notion.so/pennlabs/Creating-or-Updating-a-Member-Page-f896d85568b848c594e005708f8cb1a6